### DISCUSSION TITLE
Localize user interface: Support German

### DISCUSSION THREAD

**Discussion by burghoffdavid** [GitbookIO/community#537] [Feature Requests]
It would be great if you could add German to the Localize user interface options (only the public facing interface, not the editor). If you need any help translating, feel free to send me a list of items to be translated.

![image](https://github.com/GitbookIO/community/assets/37399963/107b2f6f-4eb3-4680-a503-202a092d695c)

Best,
David

**Comment by addisonschultz** [comment]
Hey @burghoffdavid! Great suggestion - we actually have some improvements coming in this area 😄 stay tuned!

**Reply by burghoffdavid** [reply]
Glad to hear! Do you have a rough estimation when new translations will be shipped?

**Reply by addisonschultz** [reply]
I don't have an actual date - but hoping our team can ship some work around this before the end of the month!

**Reply by burghoffdavid** [reply]
Any news on this? :)

**Reply by addisonschultz** [reply]
Hi @burghoffdavid ! Thanks for pinging - this slipped my mind, so apologies! 

We've recently open sourced the rendering portion of our app - including our translations of the UI! We don't have German yet, so it would be a great addition (and help) if you wanted to make a pull request 😄  

You can find the repository here: https://github.com/GitbookIO/gitbook?tab=readme-ov-file#translations

We also have a contributing guide here! https://github.com/GitbookIO/gitbook/blob/main/.github/CONTRIBUTING.md

**Reply by burghoffdavid** [reply]
Awesome! I will open a PR soon and link it here.

**Reply by addisonschultz** [reply]
Feel free to tag me as a reviewer 😄

**Reply by burghoffdavid** [reply]
Could not add you as a reviewer, left a tagged comment :) 
https://github.com/GitbookIO/gitbook/pull/2317
